### ISSUE TITLE
Unidentified Sourcegraph-like image

### ISSUE THREAD

**Issue by rvantonder** [Andoryuuta/NotSourcegraph#3]
Excuse me sir I ran into this in the wild, can you tell me if it's a Sourcegraph?

![image](https://user-images.githubusercontent.com/888624/71132066-bb180e80-21b3-11ea-865a-3ba08cb6e8eb.png)

**Comment by emidoots** [comment]
I cannot, for legal reasons, disclose my sources - but:

![image](https://user-images.githubusercontent.com/3173176/90301842-ff4cd300-de56-11ea-8576-301f14e4f705.png)

Maybe a fork is in order..

**Comment by rvantonder** [comment]
Thanks, this is sufficient. I think I need to rewrite this application in Rust.

**Comment by emidoots** [comment]
+1

**Comment by Andoryuuta** [comment]
@rvantonder and @slimsag,

I apologize for the belated response to this incident. This issue is deeply concerning to us, and not something that we have taken lightly.

We had originally planned to take the image you attached, run it through our NotSG algorithm, and state if it was a Sourcegraph or not. However, after much deliberation within our team, we decided it would be disingenuous to give a result from our software without physically testing it in the real world with a webcam.

As we did not have any "THE ASTERISK COLLECTIVE" merchandise to test with, it was decided that we would need to design a way to test digital logos/images, but with real-world results from a webcam.

Over the past ~7 months, we have been developing a state-of-the-art light-controlled environment for real-world dynamic unit testing. It is now complete with our new SoftScreen Diffusion Display® technology, which is used to display images of Sourcegraph (and not Sourcegraph) while simulating the softened edges of fabrics/felts (as they would be if printed on merchandise). A recent test in this environment can be seen below:

(_This is definitely not just a flashlight + tablet with a piece of paper taped on to get rid of screen glare_)
![Softscreen technology demonstration](https://user-images.githubusercontent.com/8594162/90305239-7ce00580-de8e-11ea-92a0-5acf9046c557.gif)

You might notice a few detection issues in the test video above. Shortly after finishing the testing environment, we discovered that our _NotSG_ algorithm has many false positives (and negatives) that were not present in our initial presentation of _NotSG_ (due to a lack of testing). We sincerely apologize for any confusion resulting from our lack of prior testing, however, we are not liable for any damages caused, as is stated in the MIT license.

Now, to directly answer your question: **Despite not being _the_ Sourcegraph, "THE ASTERISK COLLECTIVE" does **appear** to be _a_ Sourcegraph (according to the NotSG algorithm).** This is an issue, but likely not something we can correct at this point.
![image](https://user-images.githubusercontent.com/8594162/90305524-c4b45c00-de91-11ea-9168-8582dfdac3c0.png)

Unfortunately, due to expenses incurred during the R&D of our real-world testing environment here at NotSG, we have used up all of our VC funding (three dollars and twenty-seven cents) and will be unable to continue development on the "NotSourcegraph" algorithm any further.

A Rust rewrite sounds like a great endeavor, and we wish you luck!
